mgmt/rest/v1/rbody/error.go

```diff
@@ -47,11 +47,11 @@ func FromSnapError(pe serror.SnapError) *Error {
 func FromSnapErrors(errs []serror.SnapError) *Error {
 	fields := make(map[string]string)
 	var msg string
-	for i, err := range errs {
+	for _, err := range errs {
 		for k, v := range err.Fields() {
-			fields[fmt.Sprintf("%s_err_%d", k, i)] = fmt.Sprint(v)
+			fields[fmt.Sprintf("%s", k)] = fmt.Sprint(v)
 		}
-		msg = msg + fmt.Sprintf("error %d: %s ", i, err.Error())
+		msg = msg + fmt.Sprintf("%s", err.Error())
```

Comment:
Previously error message was without this additional string, so that is why I removed it. But I see that it could stay, so I will revert this.